vizro-core/docs/pages/user_guides/navigation.md

```diff
@@ -154,3 +154,320 @@ If you want to deviate from the default title `SELECT PAGE` and instead provide
+Currently available selectors are [`NavBar`][vizro.models.NavBar] and [`Accordion`][vizro.models.Accordion]. If `selector` is not defined it defaults to `Accordion`.
+
+If you want to add icons to your dashboard, you have to provide [`NavBar`][vizro.models.NavBar] model
+
+   ```
+    Dashboard(
+        pages=[page_1, page_2],
+        navigation=Navigation(
+            pages=["My first page", "My second page"]
+            selector=NavBar()
+        )
+    )
+   ```
+
+!!! example "Adding navigation icons"
+    === "app.py"
+        ```py
+        from vizro import Vizro
+        import vizro.plotly.express as px
+        import vizro.models as vm
+
+        iris = px.data.iris()
+
+        page_1 = vm.Page(
+                    title="My first page",
+                    components=[
+                        vm.Graph(id="scatter_chart", figure=px.scatter(iris, x="sepal_length", y="petal_width", color="species")),
+                    ],
+        )
+        page_2 = vm.Page(
+                    title="My second page",
+                    components=[
+                        vm.Graph(id="line_chart", figure=px.line(iris, x="sepal_length", y="petal_width", color="species")),
+                    ],
+                )
+
+        dashboard = vm.Dashboard(
+                        pages=[page_1, page_2],
+                        navigation=vm.Navigation(pages=["My first page", "My second page"], selector=vm.NavBar())
+                    )
+
+        Vizro().build(dashboard).run()
+        ```
+
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to register data connector in Data Manager and parse yaml configuration
+        # See from_yaml example
+        pages:
+          - components:
+              - figure:
+                  _target_: scatter
+                  data_frame: iris
+                  x: sepal_length
+                  y: petal_width
+                  color: species
+                id: scatter_chart
+                type: graph
+            title: My first page
+        - components:
+             - figure:
+                  _target_: line
+                  data_frame: iris
+                  x: sepal_length
+                  y: petal_width
+                  color: species
+                id: line_chart
+                type: graph
+            title: My second page
+        navigation:
+            pages:
+             First title:
+                - My first page
+             Second title:
+                - My second page
+            selector:
+                type: navbar
+
+        ```
+    === "Result"
+        [![NavigationIcons]][NavigationIcons]
+
+    [NavigationIcons]: ../../assets/user_guides/navigation/navigation_icons.png
+
+
+## Further customization
+
+For further customizations of icons, you can always refer to the [`NavItem`][vizro.models.NavItem] reference. Some popular choices are:
```

Comment:
This come out of the blue actually, above we only talk about `NavBar` so here I am like what the heck is `NavItem` haha